### PR TITLE
ensmallen header on not-home

### PR DESCRIPTION
**Pull request by theutahkate**

### CODE REVIEW

File: app/controllers/static_pages_controller.rb [hunk 1]
```diff
@@ -1,6 +1,9 @@
 class StaticPagesController < ApplicationController
 
   def about
+    if root_path
```
Comment (cherimarie) [inline app/controllers/static_pages_controller.rb:4]: No need for this `if`, I think. This controller action will always be the root path.

File: app/views/layouts/application.html.haml [hunk 2]
```diff
@@ -23,13 +23,20 @@
...
+            = link_to root_path do
+              = image_tag("quhac-facebookheader-pink.jpg")
+              %h1.sr-only
+                Queer Healthcare Accountability Coalition
+            %p.notice= notice
+            %p.alert= alert
+        - else
+          %header.norm
+            = link_to root_path do
+              = image_tag("quhac-facebookheader-pink.jpg", :alt => "")
```
Comment (cherimarie) [inline app/views/layouts/application.html.haml]: We probably want the %h1 here as well, right?

File: app/assets/stylesheets/sitewide.css.scss [hunk 3]
```diff
@@ -3,9 +3,23 @@ body {
   font-family: "Open Sans", Arial, sans-serif;
 }
 
-header img {
```
Comment (cherimarie) [inline app/assets/stylesheets/sitewide.css.scss:6]: Could we namespace `.norm` and `.home` as `header.norm` and `header.home`? Doubt we'll reuse the class name "norm", but might with "home".

**Comment by cherimarie** [comment]
This looks great!!! I checked it in Chrome, Firefox, and Safari.

Couple comments above, and I noticed it looks different on /providers than other pages (no underline, less room under drawing) when the screen is narrow- screenshots attached. 
screen shot 2016-04-10 at 9 02 04 am
screen shot 2016-04-10 at 9 02 14 am

File: app/views/layouts/application.html.haml (hunk 2 above)
Comment (theutahkate) [inline app/views/layouts/application.html.haml]: I removed it bc it seems like if we have the same h1 for each view, then the h1 doesn't differentiate one view from another. Do we want that?

Comment (cherimarie) [inline app/views/layouts/application.html.haml]: Yeah, the h1 is the same on both views, but right now it's only being rendered in the `if @home` block. We could just extract it from the `if/else` block entirely, and maybe the flash notices, as well.

**Pull request merged**